internal/supervisor/cluster_installation.go

```diff
@@ -244,5 +247,12 @@ func (s *ClusterInstallationSupervisor) checkReconcilingClusterInstallation(clus
 	}
 
 	logger.Info("Cluster installation finished reconciling")
+	s.metrics.ClusterInstallationCreationDurationHist.Observe(
+		elapsedTimeInSeconds(clusterInstallation.CreateAt))
```

Comment:
Cluster installations end up going into reconciling state when they are updated which means this metric will expose incorrect data in those circumstances. I am not sure of a great way around this issue without creating new states which could have other side effects if we aren't careful.

Reply:
Maybe we could lookup the installation to know whether to send the new metric data point, but that seems somewhat heavy for this. I will keep thinking on this.

Reply:
Oh, hmm, I was worried there might be an issue of this variety

Reply:
I decided to go ahead and remove the ClusterInstallation metrics from this PR. They can be added at a later time, when we determine the best approach for gathering them, whether it be adding more metadata to the model and the store, or something else.